cibyl/models/ci/base/system.py

```diff
@@ -61,7 +62,16 @@ class System(Model):
         'queried': {
             'attr_type': bool,
             'arguments': []
-        }
+        },
+        'features': {
+            'attr_type': Feature,
+            'attribute_value_class': AttributeDictValue,
+            'arguments': [
+                Argument(name='--feature', arg_type=str,
```

Comment:
What do you think about using "--features" instead of "--feature"? since it suppose to support more than one feature?